e2e/runner/bitcoin.go

```diff
@@ -286,6 +296,57 @@ func (r *E2ERunner) SendToTSSFromDeployerWithMemo(
 	return txid, nil
 }
 
+func (r *E2ERunner) InscribeToTSSFromDeployerWithMemo(
+	amount float64,
+	inputUTXOs []btcjson.ListUnspentResult,
+	memo []byte,
+) (*chainhash.Hash, error) {
+	builder := InscriptionBuilder{sidecarUrl: "http://bitcoin-node-sidecar:8000", client: http.Client{}}
+
+	address, err := builder.GenerateCommitAddress(memo)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
```suggestion
	require.NoError(r, err)
```
Can we do the same with all error handling in this function?
Note: I also realize we don't do it in other function in this file, we will do a refactoring to change it in the future

Reply:
done!